src/lib/serviceRoutinesV2/putEntityAttributeValue.cpp

```diff
@@ -79,6 +81,15 @@ std::string putEntityAttributeValue
     }
   }
 
+  if (ciP->httpStatusCode == SccConflict)
+  {
+    ErrorCode   ec("TooManyResults", "There is more than one entity that match the update. Please refine your query.");
```

Comment:
Ok with the 1st, 2nd and 5th.

However, 3rd and 4th should stay as they are now in my opinion, given they are not responses towards the user but an "assment" of what CB is doing internally to solve a conflictive situation.